docs/deployment/troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+
+- `AppServiceConsoleLogs`: `stdout` and `stderr` coming from the container
+- `AppServiceHTTPLogs`: requests coming through App Service
+- `AppServicePlatformLogs`: deployment information
+
+For some pre-defined queries, click `Queries`, then `Group by: Query type`, and look under `Query pack queries`.
+
+### [Azure Monitor Logs](https://docs.microsoft.com/en-us/azure/azure-monitor/logs/data-platform-logs)
+
+[Open the `Logs` for the slot you are interested in.](https://docs.google.com/document/d/11EPDIROBvg7cRtU2V42c6VBxcW_o8HhcyORALNtL_XY/edit#heading=h.n0oq4r1jo7zs)
+
+The following [tables](https://docs.microsoft.com/en-us/azure/azure-monitor/app/opencensus-python#telemetry-type-mappings) are likely of interest:
+
+- `requests`
+- `traces`
+
+In the latter two, you should see recent log output. Note [there is some latency](https://docs.microsoft.com/en-us/azure/azure-monitor/logs/data-ingestion-time).
+
+See [`Failures`](https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-exceptions#diagnose-failures-using-the-azure-portal) in the sidebar (or `exceptions` under `Logs`) for application errors/exceptions.
+
+### Live tail
+
+After [setting up the Azure CLI](#making-changes), you can use the following command to [stream live logs](https://docs.microsoft.com/en-us/azure/app-service/troubleshoot-diagnostic-logs#in-local-terminal):
+
+```sh
+az webapp log tail --resource-group RG-CDT-PUB-VIP-CALITP-P-001 --name AS-CDT-PUB-VIP-CALITP-P-001 --slot dev 2>&1 | grep -v /healthcheck
+```
+
+`--slot dev` can be removed for production or changed for a different slot.
+
+### SCM
+
+<https://as-cdt-pub-vip-calitp-p-001-dev.scm.azurewebsites.net/api/logs/docker>
+
+## Terraform lock
+
+[General info](https://developer.hashicorp.com/terraform/language/state/locking)
+
+If Terraform commands fail (locally or in the Pipeline) due to an `Error acquiring the state lock`:
+
+1. Check the `Lock Info` for the `Created` timestamp. If it's **in the past ten minutes** or so, that probably means Terraform is still running elsewhere, and you should wait (stop here).
+1. **Are any [Pipeline runs](https://calenterprise.visualstudio.com/CDT.OET.CAL-ITP/_build?definitionId=828) stuck?** If so, cancel that build, and try re-running the Terraform command.
+1. **Do any engineers have a Terrafrom command running locally?** You'll need to ask them. For example: They may have started an `apply` and it's sitting waiting for them to [approve](https://developer.hashicorp.com/terraform/cli/commands/apply#automatic-plan-mode) it. They will need to (gracefully) exit for the lock to be relased.
+1. **If none of the steps above identified the source of the lock**, and especially if the `Created` time is more than ten minutes ago, that probably means the last Terraform command didn't release the lock. You'll need to grab the `ID` from the `Lock Info` output and [force unlock](https://developer.hashicorp.com/terraform/language/state/locking#force-unlock).
```

Comment:
New stuff.

Reply:
Could you add a link to this [issue](https://github.com/cal-itp/benefits/issues/960) that has helpful links to examples of Pull Requests with this error and the exact messages ?

Reply:
The Benefits Azure DevOps Project is configured to only retains the Pipeline runs for 30 days, so unfortunately those won't be visible. I can link to the issue if you feel strongly; anything that I should have copied over but didn't?